subworkflows/local/prepare_genome.nf

```diff
@@ -137,19 +154,15 @@ workflow BUILD_INDICES {
     }
 
     emit:
-        bwa                           = result_bwa
-        bwa_version                   = version_bwa
-        dbsnp_tbi                     = result_dbsnp_tbi
-        dict                          = result_dict
-        fai                           = result_fai
-        gatk_version                  = version_gatk
-        germline_resource_tbi         = result_germline_resource_tbi
-        intervals                     = result_intervals
-        known_indels_tbi              = result_known_indels_tbi.collect()
-        msisensorpro_scan             = result_msisensorpro_scan
-        msisensorpro_scan_version     = version_msisensorpro_scan
-        pon_tbi                       = result_pon_tbi
-        samtools_version              = version_samtools
-        tabix_version                 = version_tabix
-        target_bed_gz_tbi             = result_target_bed
+        bwa                   = result_bwa
+        dbsnp_tbi             = result_dbsnp_tbi
+        dict                  = result_dict
+        fai                   = result_fai
+        germline_resource_tbi = result_germline_resource_tbi
+        intervals             = result_intervals
+        known_indels_tbi      = result_known_indels_tbi.collect()
+        msisensorpro_scan     = result_msisensorpro_scan
+        pon_tbi               = result_pon_tbi
+        target_bed_gz_tbi     = result_target_bed
+        versions              = ch_versions.ifEmpty(null) // channel: [ versions.yml ]
```

Comment:
why send `null` into the channel here?

Reply:
Copied that over from the rnaseq pipeline